feeadjuster/feeadjuster.py

```diff
@@ -21,7 +28,40 @@ def get_ratio(our_percentage):
     return 50**(0.5 - our_percentage)
 
 
+def get_ratio_hard(our_percentage):
+    """
+    Return value is between 0 and 20: 0 -> 20; 0.5 -> 1; 1 -> 0
+    """
+    our_percentage = min(1, max(0, our_percentage))
```

Comment:
Nit: can you add this safety check also in the `get_ratio` and `get_ratio_soft` methods?

Reply:
I rarely use squash and rebase. I hope this is what we wanted. :)

Reply:
Yep. thanks